conda-forge.yml

```
@@ -4,5 +4,8 @@ conda_forge_output_validation: true
 github:
   branch_name: main
   tooling_branch_name: main
+noarch_platforms:
+  - linux_64
+  - win_64
 provider:
   linux_aarch64: default
```

Let's add `linux_ppc64le` here to ensure that platform is represented

```suggestion
provider:
  linux_aarch64: default
  linux_ppc64le: default
```